api/all/src/main/java/io/opentelemetry/api/internal/ImmutableSpanContext.java

```diff
@@ -54,6 +54,8 @@ private static AutoValue_ImmutableSpanContext createInternal(
    * @param traceFlags the trace flags of the {@code SpanContext}.
    * @param traceState the trace state for the {@code SpanContext}.
    * @param remote the remote flag for the {@code SpanContext}.
+   * @param skipIdValidation pass true to force the {@code SpanContext} to be valid, even if the
+   *     trace ID or span ID are not.
```

Comment:
IIRC this was an optimization, maybe

> pass true to skip validation of trace ID and span ID as an optimization in cases where they are known to have been already validated

Reply:
Where's that quote from? Maybe you intended it to be a code suggestion?

Reply:
ya, I was trying to avoid one confusion of adding a code suggestion that spotless would complain about, but created another confusion 😂

Reply:
It's all good. I took your suggestion....thanks.